uint8 Robot
---
bool status